packages/eslint-plugin-clarity-adoption/docs/rules/no-clr-input.md

```diff
@@ -0,0 +1,25 @@
+# Disallows usage of Clarity Angular input (no-clr-input)
+
+The use of Clarity Angular input is discouraged. Use Clarity Core input instead.
+
+## Rule Details
+
+Examples of **incorrect** code for this rule:
+
+```html
+<clr-input-container>
+  <label>input</label>
+  <input clrInput />
+</clr-input-container>
+```
+
+Examples of **correct** code for this rule:
+
+```html
+<cds-input-group>
+  <cds-input>
+    <label>input</label>
+    <input type="text" />
+  </cds-input>
+</cds-input-group>
```

Comment:
`cds-input-group` is only required when using 2 or more inputs as a single inline input.

https://clarity.design/storybook/core/?path=/story/forms-input-groups--page